content/blog/testing-1-2-3.md

```diff
@@ -5,7 +5,7 @@ author: stpeter
 categories: ["Miscellaneous"]
 ---
 
-One of the [roadmap](https://xmpp.org/xsf/roadmap.shtml) items we discussed at the [XMPP Summit](http://blog.xmpp.org/?p=12) in late February was the need to certify that XMPP clients and servers comply with our protocol specifications. 
+One of the [roadmap](/about/xsf/roadmap) items we discussed at the XMPP Summit in late February was the need to certify that XMPP clients and servers comply with our protocol specifications. 
```

Comment:
Another WordPress `?p=(number)` link, looks like the current URL is https://xmpp.org/2007/02/brussels-report/

Reply:
Thanks! Found some more and fixed them as well (thank you, internet archive).